serverless/src/submitSwodlrOrder/retrieveCMRGranules.js

```diff
@@ -12,11 +12,9 @@ export const retrieveCMRGranules = async ({
   accessToken,
   granuleParams
 }) => {
-  const { concept_id: granuleIds } = granuleParams
-
   const granuleResponse = await axios({
     url: `${getEarthdataConfig(earthdataEnvironment).cmrHost}/search/granules.json`,
-    params: { concept_id: granuleIds },
+    params: granuleParams,
```

Comment:
A change like this should have broken a test. Because it didn't, that means we aren't testing the parameters we are sending CMR. Whatever tests we might have need to be improved to verify we are sending the correct data to CMR